docs/source/fitting/fitfunctions/CrystalFieldSpectrum.rst

```diff
@@ -17,6 +17,18 @@ Any other parameters that need fixing has to be tied explicitly. Peak centres an
 field parameters with the :ref:`CrystalFieldPeaks <func-CrystalFieldPeaks>` function. Any other peak parameter can be set using
 the "f-index-dot-name" syntax (see :ref:`CompositeFunction <func-CompositeFunction>` for more details).
 
+The fit can also consider external and molecular magnetic fields (eg. `BextX=10`, `BmolY=12`), measured in Tesla.
```

Comment:
I think for completeness if you have this section, then you need to put in text about the crystal field Hamiltonian too, and then it gets too long. It would be better to just have a link to the Crystal Field Concepts page.

Reply:
I have added the link, without any explicit mention to external or molecular fields. Is this what you had in mind or should I add a short sentence mentioning external and molecular fields?